packages/terra-dropdown-button/src/terra-dev-site/doc/DropdownButton/Button.3.doc.jsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable import/no-webpack-loader-syntax, import/first, import/extensions, import/no-duplicates, import/no-unresolved */
+import React from 'react';
+import DocTemplate from 'terra-doc-template';
+import ReadMe from '../../../../docs/Button.md';
+import { name } from '../../../../package.json';
+
+// Component Source
+import ButtonSrc from '!raw-loader!../../../../src/Button';
+
+// Example Files
+import ButtonExamples from '../example/Button';
+import ButtonExampleSrc from '!raw-loader!../../../../src/terra-dev-site/doc/example/Button.jsx';
+
+const DocPage = () => (
```

Comment:
Can we drop this page and pull in the button doc to both the dropdown doc page and the split button doc page.